src/components/Elements.test.tsx

```diff
@@ -283,7 +283,7 @@ describe('Elements', () => {
     const {result} = renderHook(() => useStripe());
 
     expect(result.error && result.error.message).toBe(
-      'Could not find Elements context; You need to wrap the part of your app that calls useStripe() in an <Elements> provider.'
+      'Cannot find either Elements or CustomCheckout context; You need to wrap the part of your app that calls useStripe() in either <Elements> or <CustomCheckoutProvider> provider.'
```

Comment:
People will start running into this error today. We shouldn't expose a beta shape in GA error messages.

Reply:
Fixed in https://github.com/stripe/react-stripe-js/pull/439/commits/d16662ad985721528520d1cb959319b1cfe00fcd